berth_reservations/tests/utils.py

```diff
@@ -84,3 +84,8 @@ def assert_invalid_enum(field_name, expected_value, executed):
     assert expected_value in errors
     assert field_name in errors
     assert f'Expected type "{expected_value}"' in errors
+
+
+def assert_model_doesnt_exist(model, executed):
```

Comment:
you already have the same util a few lines above 🤔

Reply:
🤦‍♀ 
TIL: naming is important.

Reply:
somehow, this sneaked back in 😅

Reply:
WHAT?